docs/source/datasets/index.rst

```diff
@@ -43,60 +43,67 @@ The following columns are included in this dataset:
 
    * - Attribute Name
      - Column Name
-     - Notes    
+     - Notes
    * - Unique simulant ID
      - :code:`simulant_id`
      - Not affected by noise functions; intended use is "ground truth" for testing and validation.
    * - Unique household ID
      - :code:`household_id`
      - Not affected by noise functions; intended use is "ground truth" for testing and validation; consistent across all
-       datasets. 
+       datasets.
    * - First name
      - :code:`first_name`
-     - 
+     -
    * - Middle initial
      - :code:`middle_initial`
-     - 
+     -
    * - Last name
      - :code:`last_name`
-     - 
+     -
    * - Age
-     - :code:`age` 
-     - Rounded down to an integer. 
+     - :code:`age`
+     - Rounded down to an integer.
    * - Date of birth
      - :code:`date_of_birth`
      - Formatted as YYYY-MM-DD.
    * - Physical address street number
      - :code:`street_number`
-     - 
+     -
    * - Physical address street name
      - :code:`street_name`
-     - 
+     -
    * - Physical address unit number
      - :code:`unit_number`
-     - 
+     -
    * - Physical address city
-     - :code:`city` 
-     -    
+     - :code:`city`
+     -
    * - Physical address state
-     - :code:`state`  
-     - 
+     - :code:`state`
+     -
    * - Physical address ZIP code
      - :code:`zipcode`
-     - 
+     -
+   * - Housing type
+     - :code:`housing_type`
+     - Possible values for housing type are "Household" for an individual
+       household, or one of six different types of group quarters. The types of
+       instiutional group quarters are "Carceral", "Nursing home", and "Other
+       institutional". The types of noninstitutional group quarters are
+       "College", "Military", and "Other noninstitutional".
    * - Relationship to reference person
-     - :code:`relationship_to_reference_person` 
+     - :code:`relationship_to_reference_person`
      - Possible values for this indicator include:
```

Comment:
I noticed that the "relationship to reference person" column for the decennial census is missing some possible values, namely: "Opposite-sex spouse"; "Opposite-sex unmarried partner"; "Same-sex spouse"; "Same-sex unmarried partner"

I did not update these values in this pull request in case we want to make this correction as a hotfix instead. So if this PR gets merged to `develop`, the documentation for census and ACS will not mach until we implement the hotfix and merge it to `develop`. Alternatively, I could just make the change here, and we could not worry about making the correction in a hotfix. @aflaxman any preference?

Reply:
Keep it simple, just make the changes here.